packages/react/src/router/v6/types.ts

```diff
@@ -2,6 +2,24 @@ import type { ReactElement, ReactNode } from 'react';
 
 import type { ReactRouterLocation } from '../types';
 
+interface IndexRouteObjectV6DataRouter {
+  caseSensitive?: boolean;
+  children?: undefined;
+  element?: React.ReactNode | null;
+  index?: true;
```

Comment:
I think `index` shouldn't be optional here?

Reply:
Good catch thank you 🙏